var/spack/repos/builtin/packages/serenity/package.py

```diff
@@ -40,6 +50,12 @@ class Serenity(CMakePackage):
 
     extends("python", when="+python")
 
+    patch(
+        "https://github.com/qcserenity/serenity/commit/af9f76d013e240d971337a467a03640cb9aabfb7.patch",
```

Comment:
Package audit provided a reminder for patch URLs

```suggestion
        "https://github.com/qcserenity/serenity/commit/af9f76d013e240d971337a467a03640cb9aabfb7.patch?full_index=1",
```